src/literal.h

```diff
@@ -233,7 +234,9 @@ class Literal {
     assert(type.isNullable());
     return Literal(type);
   }
-  static Literal makeFunc(Name func) { return Literal(func.c_str()); }
+  static Literal makeFunc(Name func, Type type = Type::funcref) {
+    return Literal(func.c_str(), type);
```

Comment:
Is this `c_str()` just implicitly made back into a `Name`? I don't see a `Literal` constructor that takes `char*`.

Reply:
Good point, looks like that `.c_str()` was unnecessary before, fixing.